lib/inputguess.rb

```diff
@@ -1,44 +1,28 @@
 class InputGuess
-  attr_reader :user_input, :valid_guess
-
-  def initialize
-    @valid_guess = false
-    @user_input = ""
-  end
-
-  def input_guess
-    until @valid_guess
-      print "Make a guess using available colors (R, O, Y, G, B, P): "
-      @user_input = clean_guess(gets.chomp)
-      if validate_length(@user_input)
-        validate_colors(@user_input)
+  def guess(colors)
+    valid_guess = false
+    until valid_guess
+      print "Make a guess using available colors #{colors}: "
+      user_input = clean_guess(gets.chomp)
+      return user_input if user_input == "QUIT" || user_input == "RESTART"
```

Comment:
Looks like we have some stuff in this `guess` function that we might want to test. Perhaps we can pull this out into a testable function?